internal/generator/vector/sources_to_inputs.go

```diff
@@ -35,6 +35,7 @@ var (
 	AddLogTypeApp   = fmt.Sprintf(".log_type = %q", logging.InputNameApplication)
 	AddLogTypeInfra = fmt.Sprintf(".log_type = %q", logging.InputNameInfrastructure)
 	AddLogTypeAudit = fmt.Sprintf(".log_type = %q", logging.InputNameAudit)
+	AddInfraType    = fmt.Sprintf(".infra_type=%q", "journal")
```

Comment:
Is this field new in the data model?  I don't recall this existing before

Reply:
I added this field, since journal logs can easily get lost in the infrastructure logs

Reply:
removed it